python/dask_cudf/dask_cudf/groupby.py

```diff
@@ -47,11 +60,11 @@ def __getitem__(self, key):
         g._meta = g._meta[key]
         return g
 
-    def mean(self, split_every=None, split_out=1):
```

Comment:
So, we are removing `mean`, or am I just getting confused by the diff?  I recall adding this definition because the upstream `_GroupbBy.mean` was quite slow. Is this no longer the case?

Reply:
My bad I didn't realize that was the reasoning for including those here, I just saw that tests passed without them and assumed they were no longer necessary - I will add back